dev/buildtool/base_metrics.py

```
@@ -353,6 +353,7 @@ def time_call(self, name, labels, label_func,
       except Exception as ex:
         logging.exception('label_func failed with %s', ex.message)
         raise ex
+      raise
```

Intentional?

---

yea, this was eating exceptions before